addon-test-support/reorder.js

```diff
@@ -0,0 +1,21 @@
+import { findAll } from '@ember/test-helpers';
+import drag from './drag';
+
+export default async function reorder(mode, itemSelector, ...resultSelectors) {
+  const items = findAll(itemSelector);
```

Comment:
I think this needs to be inside the `reduce`; otherwise it doesn't reflect the changed order after each iteration. (At least, that's something I had to change to make it work in one of my apps.)